index.html

```diff
@@ -50,126 +50,424 @@ <h1>Hey There.
         </section>
 
         <section class="section-two">
-            <div class="section-title">
-                <h2>My Recent Works</h2>
-                <hr>
-            </div>
+          <div class="section-title">
+            <h2>My Recent Works</h2>
+            <hr>
+          </div>
         
-            <div class="recent-work">
-                <div class="image-1">
-                  <img src="./images/section-1.png" alt="">
-                </div>
-                <div class="section-one">
-                    <h2>Multi-Post Stories</h2>
-                    <p>
-                        A daily selection of privately personalized reads;
-                        no accounts or sign-ups required. 
-                        has been the industry's standard dummy text ever since the 1500s, 
-                        when an unknown printer took a standard dummy text.
-                    </p>                    
-                    <ul class="grid-container">
-                        <li class="grid-item1"><a href="#">css</a></li>
-                        <li class="grid-item2"><a href="#">html</a></li>
-                        <li class="grid-item3"><a href="#">bootstrap</a></li>
-                        <li class="grid-item4"><a href="#">Ruby</a></li>
-                    </ul>
-                    <button name="button" type="submit" class="btn-1">See Project</button>
-                </div>
+          <div class="recent-work">
+            <div class="image-1">
```

Comment:
- [ ] Please make sure that your mobile version still work even after I change back to it 
I think this problem comes from media queries used poorly, also make sure to 
![image](https://user-images.githubusercontent.com/74744787/160025268-42aa05ac-e6bb-44a8-bd7d-1f93adf500b2.png)